ca/ocsp_test.go

```diff
@@ -97,8 +97,8 @@ func TestOcspNoEmptyLines(t *testing.T) {
 	test.AssertDeepEquals(t, log.GetAll(), []string{})
 }
 
-// If the maxLogLen is 0, don't log anything.
-func TestOcspLogMaxLenZeroMeansNoLog(t *testing.T) {
+// If the maxLogLen is 0, log everything immediately.
+func TestOcspLogMaxLenZeroMeansLogImmediately(t *testing.T) {
```

Comment:
I think that this will be confusing to future readers. Within an `ocspLogQueue` proper, yes, a `maxLogLen` of zero means "log immediately", but so does any length less than `ocspSingleLogEntryLen`. Meanwhile, in ca.go, a `maxLogLen` of 0 means "don't create an `ocspLogQueue` at all". I think that this test should be updated to be "some value between 0 and `ocspSingleLogEntryLen`", and that ca.go should have a test added which asserts that no queue is created when the `maxLogLen` is zero.